src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -866,6 +866,14 @@ tests:
               - name: "OffWaitTime"
                 value: 300
 
+    - label: "Wait 10000ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 10000
```

Comment:
This PR is missing the code regeneration. If no other files than the YAML are modified, it means the tests are not actually modified. Please run `scripts/tools/zap_regen_all.py --type tests`.

Reply:
Thx!
will do so...

Reply:
Done.